docs/docs/integrations/platforms/aws.mdx

```diff
@@ -197,7 +197,26 @@ See a [usage example](/docs/integrations/vectorstores/documentdb).
 ```python
 from langchain.vectorstores import DocumentDBVectorSearch
 ```
+### Amazon MemoryDB 
+[Amazon MemoryDB](https://aws.amazon.com/memorydb/) MemoryDB is a durable, in-memory database service that delivers ultra-fast performance. MemoryDB is compatible with Redis OSS, a popular open source data store, 
+enabling you to quickly build applications using the same flexible and friendly Redis OSS APIs, and commands that they already use today. 
 
+AWS offers services for computing, databases, storage, analytics, and other functionality. For an overview of all AWS services, see [Cloud Computing with Amazon Web Services](https://aws.amazon.com/what-is-aws/).
+
```

Comment:
Is this necessary? Better to be precise about the relevant service, than promoting AWS.